run_unittests.py

```diff
@@ -1556,5 +1556,15 @@ def test_subdir(self):
         self.assertEqual(s2, self.read_contents('sub2/meson.build'))
 
 
+def unset_envs():
+    # For unit tests we must fully control all commend lines
+    # so that there are no unexpected changes coming from the
+    # environment, for example when doing a package build.
+    varnames = ['CFLAGS', 'CXXFLAGS', 'CPPFLAGS']
```

Comment:
What about `FFLAGS`, `DFLAGS`, `VALAFLAGS` and everything else [out of here](https://github.com/mesonbuild/meson/blob/aa486f5bb2b962ee6f5f65cabe05bb33f92ebead/mesonbuild/environment.py#L813)? `LDFLAGS`, too?